Oqtane.Server/Managers/UserManager.cs

```diff
@@ -66,7 +66,7 @@ public User GetUser(int userid, int siteid)
                     user.SiteId = siteid;
                     user.Roles = GetUserRoles(user.UserId, user.SiteId);
                     List<Setting> settings = _settings.GetSettings(EntityNames.User, user.UserId).ToList();
-                    user.Settings = settings.Where(item => !item.IsPrivate || user.UserId == user.UserId)
+                    user.Settings = settings.Where(setting => !setting.IsPrivate && setting.EntityId == user.UserId)
```

Comment:
> user.UserId == user.UserId

This was a big mistake that might have happened accidentally. Also, I think we need to use and operator instead of or